src/components/EnterBarcode/EnterBarcodePage.tsx

```diff
@@ -28,17 +28,24 @@ const EnterBarcodePage = ({
       if (selectedPatient.length > 0) {
         // Found patient
         const {
-          collectionPointId: { id: patientCCPId },
+          collectionPointId: {
+            id: patientCCPId,
+            eventId: { id: patientEventId },
+          },
           id,
         } = selectedPatient[0];
         // Redirect to patient profile
-        history.replace(`/patients/edit/${patientCCPId}/${id}`);
+        history.replace(
+          `/events/${patientEventId}/ccps/${patientCCPId}/patients/${id}`
+        );
       } else {
         // No existing patient
-        history.replace(`/patients/new/${ccpID}/${barcode}`);
+        history.replace(
+          `/events/${eventID}/ccps/${ccpID}/patients/new/${barcode}`
+        );
       }
     }
-    history.replace(`/patients/new/${ccpID}/${barcode}`);
+    history.replace(`/events/${eventID}/ccps/${ccpID}/patients/new/${barcode}`);
```

Comment:
Not sure why this is here because when I enter an existing patient's barcode I always get redirected to this page to create a new patient

Reply:
Could you provide a screenshot of the backend view of the patient, CCP ID and event ID as well as the front-end view of this (please also include the URL at the top). I didn't run into this issue before

Reply:
![image](https://user-images.githubusercontent.com/38408276/91338542-1e2a5e00-e7a3-11ea-8002-531bd19fd3cc.png)

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/38408276/91339143-128b6700-e7a4-11ea-9b1c-9d602555c501.gif)

Reply:
Hmm I'm not sure what's going on if both u and peter aren't experiencing it

Reply:
this might be a bug with the `/manual` page - @aaronabraham311 see if you can repro and we can decide if this should be a separate ticket